util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -115,6 +115,9 @@ package object file {
         case t: Throwable => file.getAbsoluteFile
       }
 
+    def descendsFrom(parent: File): Boolean =
+      file.getParent.startsWith(parent.getAbsolutePath)
```

Comment:
yup, 100% agree. The conversions are a result of my lack of knowledge around the `vfs2` api.